usagov/vote-gov-drupal
# VOTE-2240: Main navigation menu styles - high contrast

## Jira ticket

[VOTE-2240](https://cm-jira.usa.gov/browse/VOTE-2240)

## Description

Adds high contrast variables for the main navigation component on desktop and mobile

## Deployment and testing

### QA/Testing instructions

1. lando retune
2. npm run build
4. Confirm no style regressions on main navigation menu
6. Toggle high contrast mode on
7. Confirm high contrast styles
8. Switch to mobile view
9. Confirm no style regressions and high contrast styles

## Checklist for the Developer




## Checklist for the Peer Reviewers
